client/src/components/Button/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import btnimg from '../../btn.jpg';
+
+
+class Button extends Component {
+  state = {  }
```

Comment:
You don't need to declare an empty state.  This is probably leftover from the initial setup.  Other than that, this is a very clean and organised component!